build.rs

```diff
@@ -0,0 +1,5 @@
+//!
+//! Build script for the era-compiler-solidity
+//!
+
+fn main() {}
```

Comment:
Empty build script is required to make [`rustc-link-lib` setting](https://github.com/matter-labs/era-compiler-solidity/pull/68/files#diff-9a4f3e4537ebb7474452d131b0d969d89a51286f4269aac5ef268e712be17268R24) work.

Reply:
@antonbaliasnikov please add such things to comments in the code!